common/proto/SDMS_Auth.proto

```diff
@@ -907,14 +907,13 @@ message RepoCreateRequest
     required string             id          = 1; // ID of repo
     required string             title       = 2; // Title
     optional string             desc        = 3; // Description
-    optional string             domain      = 5; // RESERVED
-    required string             path        = 6; // Path to storage directories
-    optional string             exp_path    = 7; // RESERVED
```

Comment:
I'm not following what "exp_path" is actually for from the Globus 4?

Reply:
It was intended to be used in conjunction with the FUSE driver to tell end-users what the local path to the controlled data in the repo was (so they could open it directly). None of this will work going forward if we want to support non-posix storage options.